packages/ui-react/src/components/Header/Header.module.scss

```diff
@@ -81,7 +81,7 @@
     height: 36px;
     min-height: 36px;
     margin: 0 6px;
-    padding: 0 calc(#{variables.$base-spacing} / 2);
+    padding: 22px calc(#{variables.$base-spacing} / 2);
```

Comment:
What is this 22px based on? Can we perhaps take it from a variable, that is also used for the header heights?

Reply:
I've refactored this piece of code to implement the following calculation: `calc(variables.$header-height / 2.5)`, it uses the height of the header to position it